ex5/ex5.c

```diff
@@ -10,13 +10,49 @@
 
 #define MSGSIZE 16
 
-char* msg1 = "hello world #1";
-char* msg2 = "hello world #2";
-char* msg3 = "hello world #3";
+char *msg1 = "hello world #1";
```

Comment:
Good job moving these asterisks over. I was telling one of your classmates that it's funny how you can tell who the author of code is by something as simple as a single whitespace.